test/parallel/test-webcrypto-digest.js

```diff
@@ -168,3 +168,9 @@ async function testDigest(size, name) {
 
   await Promise.all(variations);
 })().then(common.mustCall());
+
+(async () => {
+  await assert.rejects(subtle.digest('RSA-OAEP', Buffer.alloc(1)), {
+    name: 'NotSupportedError',
+  });
+})().then(common.mustCall());
```

Comment:
nit: the async wrapper is technically not needed (although it's fine to keep it if you prefer as is)
```suggestion
assert.rejects(subtle.digest('RSA-OAEP', Buffer.alloc(1)), {
    name: 'NotSupportedError',
}).then(common.mustCall());
```